builders/relayedTxV1Builder_test.go

```diff
@@ -0,0 +1,104 @@
+package builders
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go-crypto/signing"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/ed25519"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/blockchain"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/blockchain/cryptoProvider"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/core"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/interactors"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testRelayerMnemonic     = "bid involve twenty cave offer life hello three walnut travel rare bike edit canyon ice brave theme furnace cotton swing wear bread fine latin"
+	testInnerSenderMnemonic = "acid twice post genre topic observe valid viable gesture fortune funny dawn around blood enemy page update reduce decline van bundle zebra rookie real"
+)
+
+func TestRelayedTxV1Builder(t *testing.T) {
+	proxyProvider, err := blockchain.NewElrondProxy(blockchain.ArgsElrondProxy{
+		ProxyURL:            "https://testnet-gateway.elrond.com",
+		CacheExpirationTime: time.Minute,
+		EntityType:          core.Proxy,
+	})
+	require.NoError(t, err)
+
+	netConfig, err := proxyProvider.GetNetworkConfig(context.Background())
```

Comment:
do we make a http request in every unit test? maybe hardcode the net config values needed for this test?

Reply:
forgot to remove it. done